phoenix-core/src/main/java/org/apache/phoenix/coprocessor/UngroupedAggregateRegionObserver.java

```diff
@@ -1088,9 +1100,17 @@ private RegionScanner rebuildIndices(final RegionScanner innerScanner, final Reg
             }
             innerScanner.close();
             RegionScanner scanner = region.getScanner(rawScan);
-            return new IndexRebuildRegionScanner(scanner, region, scan, env, this);
+            if (oldCoproc) {
+                return new IndexerRegionScanner(scanner, region, scan, env, this);
```

Comment:
For old coproc, we wanted to do only one version verification. Moving this inside not-raw block, wouldn't it change the strategy?

Reply:
Also, what happens to the specific check for ONLY?

Reply:
For the old indexing design, the verification other than ONLY is not supported. This was a problem before this PR too. This should be fixed in IndexTool. I will add the fix.

Reply:
Okay, thanks! I think I misread the change at line 1088.